src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -91,9 +90,13 @@ To provide a seamless experience between your extension and the browser, use the
 
 ### Themes
 
-- Use the Protocol Design System's [color palette](https://protocol.mozilla.org/fundamentals/color.html).
+Use the Protocol Design System’s [color palette](https://protocol.mozilla.org/fundamentals/color.html).
```

Comment:
I am not sure we should continue to recommend this design system. According to https://acorn.firefox.com/latest/f-a-q-g56QlqB6, we should probably recommend Acorn.

Reply:
Good point, thanks – updating relevant links